content/vi/docs/tasks/tools/install-minikube.md

```diff
@@ -0,0 +1,210 @@
+  name: tasks
+  weight: 10
+---
+
+{{% capture overview %}}
+
+Tài liệu này sẽ chỉ cho bạn các cài đặt [Minikube](/docs/tutorials/hello-minikube), một công cụ chạy một single-node Kubernetes cluster trong một máy ảo (VM) trên máy tính của bạn.
+
+{{% /capture %}}
+
+{{% capture prerequisites %}}
+
+{{< tabs name="minikube_before_you_begin" >}}
+{{% tab name="Linux" %}}
+Để kiểm tra hỗ trợ ảo hóa trên Linux, hãy chạy câu lệnh sau và xác nhận có giá trị kết quả đầu ra.
+```
+grep -E --color 'vmx|svm' /proc/cpuinfo
+```
+{{% /tab %}}
+
+{{% tab name="macOS" %}}
+Để kiểm tra hỗ trợ ảo hóa trên macOS, hãy chạy câu lệnh sau trên cửa sổ terminal của bạn.
+```
+sysctl -a | grep -E --color 'machdep.cpu.features|VMX' 
+```
+Nếu bạn nhìn thấy `VMX` ở kết quả (có màu), thì VT-x đã được enable trên máy của bạn.
+{{% /tab %}}
+
+{{% tab name="Windows" %}}
+Để kiểm tra hỗ trợ ảo hóa trên Windows 8 hay phiên bản Windows cao hơn, hãy chạy câu lệnh sau trên terminal của Window hoặc command prompt.
+```
+systeminfo
+```
+Nếu bạn nhìn thấy những thông tin đầu ra sau, tức là ảo hóa được hỗ trợ trên máy tính của bạn.
+```
+Hyper-V Requirements:     VM Monitor Mode Extensions: Yes
+                          Virtualization Enabled In Firmware: Yes
+                          Second Level Address Translation: Yes
+                          Data Execution Prevention Available: Yes
+```
+
+Nếu bạn nhìn thấy những thông tin đầu ra sau, tức là máy tính của bạn đã được cài đặt Hypervisor và bạn có thể bỏ qua những bước tiếp theo.
+```
+Hyper-V Requirements:     A hypervisor has been detected. Features required for Hyper-V will not be displayed.
+```
+
+
+{{% /tab %}}
+{{< /tabs >}}
+
+{{% /capture %}}
+
+{{% capture steps %}}
+
+# Cài đặt minikube
+
+{{< tabs name="tab_with_md" >}}
+{{% tab name="Linux" %}}
+
+### Cài đặt kubectl
+
+Đảm bảo bạn đã cài đặt kubectl. Bạn có thể cài đặt kubectl theo hướng dẫn sau tại [Install and Set Up kubectl](/docs/tasks/tools/install-kubectl/#install-kubectl-on-linux).
+
+### Cài đặt Hypervisor
+
+Nếu bạn chưa cài đặt Hypervisor, hãy cài đặt một trong những phần mềm sau đây:
+
+• [KVM](https://www.linux-kvm.org/), sử dụng QEMU
+
+• [VirtualBox](https://www.virtualbox.org/wiki/Downloads)
+
+{{< note >}}
+Minikube cũng hỗ trợ tùy chọn `--vm-driver=none` để chạy các thành phần của Kubernetes trên máy chủ chứ không phải trong VM. Sử dụng driver này yêu cầu [Docker](https://www.docker.com/products/docker-desktop) và môi trường Linux không phải là một Hypervisor. Bạn nên sử dụng cài đặt apt của docker từ [Docker](https://www.docker.com/products/docker-desktop) khi sử dụng non driver. Cài đặt snap của docker không hoạt động với minikube.
+{{< /note >}}
+
+### Cài đặt Minikube sử dụng package
+
+Có các gói *thử nghiệm* cho Minikube có sẵn; bạn có thể tìm thấy các gói Linux (AMD64) từ trang [phát hành](https://github.com/kubernetes/minikube/releases) của Minikube trên Github.
+
+Sử dụng công cụ phân phối Linux package của bạn để cài đặt gói phù hợp.
```

Comment:
Should be:
"Sử dụng các package tool của bản phân phối Linux của bạn để cài đặt package phù hợp."